### DISCUSSION TITLE
Duplicate Simplisafe Devices

### DISCUSSION THREAD

**Discussion by Lorccan** [homebridge-simplisafe3/homebridge-simplisafe3#410] [Q&A] [answered]
I had the SimpliSafe3 plugin installed and all was working fine up until a few days ago when I noticed that there were duplicates of all my contact sensors in the "Default Room" in the Home app.

I tried deleting the duplicates but they would not disappear.

I then tried removing the SimpliSafe3 bridge (I run it as a homebridge child bridge) from the Home app and was surprised to see a second one appear, so I deleted that as well. Then for good measure I removed the plugin from homebridge as well.  At this point - as expected - there were no SimpliSafe devices in the Home app.

After restarting the homebridge server, I then reinstalled the SimpliSafe3 plugin and set it to be a child bridge again. Although I had completely removed the configuration when I removed it from homebridge, there was a message that authentication had already been completed. (I didn't note the precise words.)

I went ahead and added SimpliSafe to Home app again. Everything appeared normal, until I checked the Default Room and found that I now have 2 (non responsive) duplicates of each of my sensors.

My guess is that something is being left behind when the plugin is removed, but does anyone know how I can get back to a clean install so that I don't get these ghost devices?

**Comment by shamoon** [comment, marked as answer]
I think you'll want to remove cached accessories in homebridge, e.g. https://github.com/homebridge/homebridge-config-ui-x/issues/525

**Comment by Lorccan** [comment]
That did it. Thank you!
